internal/cmd/stack/run_review.go

```diff
@@ -0,0 +1,64 @@
+package stack
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/shurcooL/graphql"
```

Comment:
do we do import ordering here?

Reply:
I'll fix this for now, but I think we need to setup golangci-lint on this repo like we do elsewhere.